lib/pymedphys/_dicom/coords.py

```diff
@@ -200,8 +202,8 @@ def coords_in_datasets_are_equal(datasets: Sequence["pydicom.dataset.Dataset"])
         True if coordinates match for all datasets, False otherwise.
     """
 
-    if not len(datasets) >= 2:
-        raise ValueError("At least two datasets must be provided for comparison")
+    if len(datasets) == 2:
+        return True
```

Comment:
This change here has me quite confused.

Reply:
The `2` was a typo (should've been `1`) but I've removed entirely now.